internal/http/services/owncloud/ocdav/ocdav.go

```diff
@@ -216,15 +218,30 @@ func wrap(sid string, oid string) string {
 	return base64.URLEncoding.EncodeToString([]byte(fmt.Sprintf("%s:%s", sid, oid)))
 }
 
+func isValidUUID(u string) bool {
```

Comment:
@wkloucek why you enforce to have an UUID?

Reply:
@labkode The ideas was to do a sanity check. But while you're asking it is nonsense to do it only here and it is not really needed too. Therefore I removed it.